src/sql/run/pgspecial.py

```diff
@@ -0,0 +1,52 @@
+try:
+    from pgspecial.main import PGSpecial
+except ModuleNotFoundError:
+    PGSpecial = None
+
+from sql import exceptions
+
+
+def handle_postgres_special(conn, statement):
+    """Execute a PostgreSQL special statement using PGSpecial module."""
```

Comment:
Do you think we can import PGSpecial only if it's called?

```suggestion
    """Execute a PostgreSQL special statement using PGSpecial module."""
    try:
    from pgspecial.main import PGSpecial
except ModuleNotFoundError:
    PGSpecial = None
```

Reply:
I'm confused. this try ... except .. is already in line 1-4. applying this won't have any effect